setup.py

```diff
@@ -31,6 +31,7 @@
             "requests_toolbelt",
             "html5lib",
             "scikit-learn",
+            "dagster-powerbi==0.0.11"
```

Comment:
do we want the pin forever? or should we let it float like the others post-merge?

Reply:
I think we can remove now -- I was pinning aggressively due to major API shifts with the definintions loader that made it pretty dicey. That should all be resolved now